install.sh

```diff
@@ -165,3 +166,5 @@ if $nonroot; then
 elif ! $without_systemd && ! $packaging; then
     systemctl --system daemon-reload
 fi
+
+echo done
```

Comment:
it was added for debugging.

Reply:
I already merged this. You can send a follow-up if you think this should be removed.